engineio/src/packet.rs

```diff
@@ -303,33 +311,35 @@ mod tests {
         let _sut = sut.unwrap_err();
         assert!(matches!(Error::IncompletePacket, _sut));
 
-        let sut = PacketId::try_from(b'0');
-        assert!(sut.is_ok());
-        assert_eq!(sut.unwrap(), PacketId::Open);
+        assert_eq!(PacketId::MessageBinary.to_string(), "b");
+
+        let sut = PacketId::try_from(b'0').unwrap();
+        assert_eq!(sut, PacketId::Open);
+        assert_eq!(sut.to_string(), "0");
 
-        let sut = PacketId::try_from(b'1');
-        assert!(sut.is_ok());
-        assert_eq!(sut.unwrap(), PacketId::Close);
+        let sut = PacketId::try_from(b'1').unwrap();
```

Comment:
Can you please make the test return a `Result` and use the try operator here? :)